src/main/java/core/basesyntax/strategy/BalanceOperationHandler.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.strategy;
+
+import java.util.Map;
+
+public class BalanceOperationHandler implements OperationHandler {
+    @Override
+    public void apply(Map<String, Integer> inventory, String fruit, int quantity) {
+        inventory.put(fruit, quantity);
+    }
```

Comment:
According to the checklist, you should avoid beginning class or method implementation with an empty line. Please remove the redundant empty line at the start of the class definition.